tests/jsthemis/test.js

```diff
@@ -175,6 +175,21 @@ describe("jsthemis", function(){
 
 describe("jsthemis", function(){
     describe("secure cell", function(){
+        describe("key generation", function(){
+            it("generates new key buffer", function(){
+                var masterKey = new addon.SymmetricKey()
+                assert.notEqual(masterKey.length, 0)
```

Comment:
check default length?

```suggestion
                assert.notEqual(masterKey.length, 0)
                assert.equal(masterKey.length, 32)
```